examples/browser-vue/src/components/IpfsInfo.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="ipfs-info">
+    <img class="ipfs-logo" alt="IPFS logo" src="../assets/logo.svg" />
+    <h1>{{ status }}</h1>
+    <h2>ID: {{ id }}</h2>
+    <h2>Agent version: {{ agentVersion }}</h2>
+  </div>
+</template>
+
+<script>
+import pify from "pify";
+
+export default {
+  name: "IpfsInfo",
+  data: function() {
+    return {
+      status: "Connecting to IPFS...",
+      id: "",
+      agentVersion: ""
+    };
+  },
+  mounted: function() {
+    this.getIpfsNodeInfo();
+  },
+  methods: {
+    async getIpfsNodeInfo() {
+      try {
+        // Await for ipfs node instance.
+        const ipfs = await this.$ipfs;
+        // Call ipfs `id` method.
+        // Returns the identity of the Peer.
+        const { agentVersion, id } = await pify(ipfs.id)();
```

Comment:
All IPFS API methods return promises if you do not pass a callback - this is probably not necessary?